css/styleguide.scss

```diff
@@ -5,4 +5,4 @@
 @import 'uswds/dist/scss/packages/uswds-layout-grid';
 @import 'uswds/dist/scss/packages/uswds-components';
 @import 'project-custom';
-@import 'uswds/dist/scss/packages/uswds-utilities';
\ No newline at end of file
+@import 'uswds/dist/scss/packages/uswds-utilities';
```

Comment:
since we're adding this, should we just `@import uswds` before the project custom?

Reply:
Not sure I follow, this showed up as green bc I added an extra line at the end of the file, but didn't change anything. We want utilities to override custom code.

Reply:
oh i see — utilities may still not override custom code, but ok as is